### DISCUSSION TITLE
Bug: Issue status filter „all“ displays only open issues

### DISCUSSION THREAD

**Discussion by andreaswilli** [community/community#45306] [Mobile]
Individual filters for open and closed issues work fine.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.97.0 (89518966)
Device: iPhone 11 (iOS 16.2)
</details>

**Comment by iskae** [comment]
Hi @andreaswilli 👋 

Unfortunately we were not able to reproduce this issue on our end 😅 Can you maybe record a video of this behaviour so we can understand what is actually happening?

**Reply by andreaswilli** [reply]
Sure

https://user-images.githubusercontent.com/17298270/214597713-6edd241f-d00b-4a30-ad2d-d02bc4a2a18a.MOV

**Reply by ghost** [reply]
> Hi @andreaswilli 👋 
> 
> 
> 
> Unfortunately we were not able to reproduce this issue on our end 😅 Can you maybe record a video of this behaviour so we can understand what is actually happening?
> 
> 

https://user-images.githubusercontent.com/95468948/215291720-d4ca71f5-5bdd-4830-87b1-281aac2ad9a0.MOV

GitHub for iOS: 1.97.0
iOS: 16.3
iPhone: 13 Pro

**Reply by iskae** [reply]
Hi @andreaswilli & @jslawler 👋 

We were able to reproduce the issue and prepared a fix for it 🎉 It should be available in next weeks release 👍 

Thanks for reporting this ❤️ Please keep the feedback coming 🙇 

Cheers!

**Reply by andreaswilli** [reply]
Awesome 🚀
Glad I could help.
